next/package.json

```diff
@@ -29,7 +29,7 @@
   },
   "dependencies": {
     "@azure/msal-node": "^3.4.0",
-    "@bratislava/component-library": "1.0.2",
+    "@bratislava/component-library": "^1.0.3",
```

Comment:
Please change to explicit "1.0.3" version, because we don't track breaking changes in the library so far.